spec/lib/duplicate_draft_deleter_spec.rb

```diff
@@ -8,20 +8,20 @@
   it "deletes duplicate editions that aren't present in Publishing API" do
     original_content_id = SecureRandom.uuid
     FactoryGirl.create(:specialist_document_edition,
-      slug: "cma-cases/a-case",
+      slug: "guidance/slug",
```

Comment:
Since this factory returns an edition of a manual *section*, I think a more realistic path would be `guidance/manual-slug/section-slug`, would it not? Same comment for other occurrences in this commit.